operators/src/processing/expression/mod.rs

```diff
@@ -43,7 +43,7 @@ pub struct ExpressionParams {
     #[serde(with = "float_with_nan")]
     pub output_no_data_value: f64, // TODO: check value is valid for given output type during deserialization
```

Comment:
we can remove this now right? We can just put 0 there, or not? I guess we really have to think about what the meaning of a pixel value is if it is masked as no data. I'd say it has no meaning and can be arbitrary.

Reply:
nope we still need it. we have to change the return type of the expression to be either Option< T > or (T, bool). If we set it to 0 the all results of the expression that are 0 eill be flaged as invalid/no_data.

Reply:
yes we should change the return type of the expression then